app/config/MicroserviceAppConfig.scala

```diff
@@ -30,76 +31,47 @@ trait AppConfig {
 
   private def loadConfig(key: String) = servicesConfig.getString(key)
 
-  override lazy val hipBusinessDetailsURL: String = servicesConfig.baseUrl("get-itsa-business-details")
-  override lazy val getItsaBusinessDetailsEnvironmentToken = s"Basic ${loadConfig("microservice.services.get-itsa-business-details.authorization-token")}"
-
   override lazy val statusDeterminationServiceURL: String = servicesConfig.baseUrl("status-determination-service")
 
   private val statusDeterminationServiceBase = "microservice.services.status-determination-service"
   override lazy val statusDeterminationServiceAuthorisationToken: String = s"Bearer ${loadConfig(s"$statusDeterminationServiceBase.authorization-token")}"
   override lazy val statusDeterminationServiceEnvironment: String = loadConfig(s"$statusDeterminationServiceBase.environment")
 
-  override lazy val taxableEntityAPI: String = servicesConfig.baseUrl("taxable-entity-api")
-  override lazy val getITSAStatusAuthorisationToken = s"Basic ${loadConfig("microservice.services.taxable-entity-api.get-itsa-status.authorization-token")}"
-
-  override lazy val hipSignUpServiceURL: String = servicesConfig.baseUrl("hip-signup-tax-year-service")
-
-  override lazy val hipSignUpServiceAuthorisationToken: String =
-    s"Basic ${loadConfig("microservice.services.hip-signup-tax-year-service.authorization-token")}"
-
-  lazy val itsaIncomeSourceURL: String = servicesConfig.baseUrl("itsa-income-source")
-  lazy val itsaIncomeSourceAuthorisationToken: String = s"Basic ${loadConfig("microservice.services.itsa-income-source.authorization-token")}"
-
   lazy val mongoUri: String = loadConfig("mongodb.uri")
 
   lazy val timeToLiveSeconds: Int = loadConfig("mongodb.timeToLiveSeconds").toInt
   lazy val sessionTimeToLiveSeconds: Int = loadConfig("mongodb.sessionTimeToLiveSeconds").toInt
   lazy val throttleTimeToLiveSeconds: Int = loadConfig("mongodb.throttleTimeToLiveSeconds").toInt
 
-  private val hipPrePopBase = "microservice.services.hip-pre-pop"
-
-  override lazy val hipPrePopURL: String =
-    servicesConfig.baseUrl("hip-pre-pop")
+  override val getHipBaseURL: String =
+    servicesConfig.baseUrl("hip")
 
-  override lazy val hipPrePopAuthorisationToken =
-    s"Basic ${loadConfig(s"$hipPrePopBase.authorization-token")}"
+  private val appClientIdForHip: String =
+    loadConfig("microservice.services.hip.creds.clientId")
 
-  private val hipItsaStatus = "hip-itsa-status"
+  private val appClientSecretForHip: String =
+    loadConfig("microservice.services.hip.creds.clientSecret")
 
-  override val hipItsaStatusURL: String =
-    servicesConfig.baseUrl(hipItsaStatus)
+  override def getHipAuthToken: String = {
+    val parts = Seq(
+      appClientIdForHip,
+      appClientSecretForHip
+    )
 
-  override val hipItsaStatusAuthorisationToken =
-    s"Basic ${loadConfig(s"microservice.services.$hipItsaStatus.authorization-token")}"
+    "Basic " + Base64.getEncoder.encodeToString(parts.mkString(":").getBytes())
```

Comment:
```suggestion
    "Basic " + Base64.getEncoder.encodeToString(parts.mkString(":").getBytes("UTF-8"))
```
I'm not 100% sure the difference, but every example I have seen uses the UTF-8 for getBytes.
Probably worth being consistent.

Reply:
Added `"UTF-8"` as suggested.